### DISCUSSION TITLE
Explaination of the functionality of xExtension-QuickCollapse

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/Extensions#173] [Q&A] [answered]
@marienfressinaud could you please explain what is the functionality of xExtension-QuickCollapse?

I installed it but cannot find anything different.

The description is "This FreshRSS extension allows to quickly change from/to collapsed articles".

Could you please tell me an example how it works?

**Comment by Frenzie** [comment]
There's a GIF here: https://github.com/FreshRSS/FreshRSS/pull/1555

**Comment by marienfressinaud** [comment, marked as answer]
I think the GIF explains it well: it just allows you to open/close all the articles at once.

**Comment by math-GH** [comment]
Thanks!
